ambry-tools/src/main/java/com.github.ambry/store/StoreCopier.java

```diff
@@ -224,7 +225,7 @@ public void close() throws IOException {
       return;
     }
     try {
-      shutDownExecutorService(scheduler, 1, TimeUnit.SECONDS);
+      shutDownExecutorService(scheduler, 5, TimeUnit.MINUTES);
```

Comment:
what is the reason behind this change?

Reply:
It is possible that at the time the `StoreCopier` is done, there are multiple index files waiting to be written to disk. It was observed (empirically) that sometimes 1s wasn't enough.